apps/prairielearn/src/pages/news_item/news_item.html.ts

```diff
@@ -0,0 +1,75 @@
+import { z } from 'zod';
+
+import { html, unsafeHtml } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { NewsItemSchema } from '../../lib/db-types.js';
+
+export const NewsItemRowSchema = NewsItemSchema.extend({
+  formatted_date: z.string(),
```

Comment:
We now have a function to do this in JS; bonus points if you use it!

https://github.com/PrairieLearn/PrairieLearn/blob/bf96affaace6514e83c62cd828ad2083749fea06/packages/formatter/src/date.ts#L12

Reply:
I had to add support for date without time, but done.